CommandLine/libEGM/gmk.cpp

```diff
@@ -916,8 +950,7 @@ std::unique_ptr<Object> LoadObject(Decoder &dec, int /*ver*/) {
       if (second == -1) break;
 
       auto event = object->add_events();
-      event->set_type(i);
-      event->set_number(second);
+      dec.postponeEventName(event->mutable_name(), i, second);
```

Comment:
It's not clear to me how long this ref lives. Adding more events may invalidate it.